utils/contexts/auth/AuthProvider.tsx

```diff
@@ -63,12 +75,33 @@ export const AuthProvider = ({
         }
         return;
       }
-      console.log(await mainUserAuthValidate(user));
-      console.log("solama");
-      // if logged in
+
+      // if logged in & has localUserData
       if (localUserData) {
+        // check if data has changed
+        // by comparing lastLoginAt on auth database and local machine
+        console.log((user.metadata as any).lastLoginAt);
+
+        console.log((localUserData?.localAuthData as any).lastLoginAt);
+
+        if (
+          (user.metadata as any).lastLoginAt !==
+          (localUserData?.localAuthData as any).lastLoginAt
+        ) {
+          // getting new data
+          // console.log("getting new data");
+          const newUserData = await fbUserGet({ email: localUserData.email });
```

Comment:
Noted for future re-login fix using server-side

Reply:
Just for formality please do answer this so i can proceed @arunstop

Reply:
Okay noted already thanks my dear variant!